src/GitHub.VisualStudio/UI/GitHubPane.cs

```diff
@@ -83,17 +88,26 @@ public void Initialize(IServiceProvider serviceProvider)
         {
             if (!initialized)
             {
-                var provider = VisualStudio.Services.GitHubServiceProvider;
-                var teServiceHolder = provider.GetService<ITeamExplorerServiceHolder>();
-                teServiceHolder.ServiceProvider = serviceProvider;
+                InitializeAsync(serviceProvider).Forget();
```

Comment:
If this async method throws, you'll never know. 
I find it's best to create an extension method on Task (we call ours `FileAndForget()`) that will attach an OnlyOnFaulted continuation (or await) on the Task and file a failure report, or display an error to the user, or whatever... something to draw attention to the failure so it's more diagnosable.